config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java

```diff
@@ -55,75 +60,71 @@ private static Set<Metric> getVespaMetrics() {
 
-        addMetric(metrics, "logd.processed.lines.count");
-        addMetric(metrics, "worker.connections.max");
-        addMetric(metrics, "endpoint.certificate.expiry.seconds");
+        addMetric(metrics, LogdMetrics.LOGD_PROCESSED_LINES.count());
 
         // Java (JRT) TLS metrics
-        addMetric(metrics, "jrt.transport.tls-certificate-verification-failures");
-        addMetric(metrics, "jrt.transport.peer-authorization-failures");
-        addMetric(metrics, "jrt.transport.server.tls-connections-established");
-        addMetric(metrics, "jrt.transport.client.tls-connections-established");
-        addMetric(metrics, "jrt.transport.server.unencrypted-connections-established");
-        addMetric(metrics, "jrt.transport.client.unencrypted-connections-established");
+        addMetric(metrics, ContainerMetrics.JRT_TRANSPORT_TLS_CERTIFICATE_VERIFICATION_FAILURES.baseName());
+        addMetric(metrics, ContainerMetrics.JRT_TRANSPORT_PEER_AUTHORIZATION_FAILURES.baseName());
+        addMetric(metrics, ContainerMetrics.JRT_TRANSPORT_SERVER_TLS_CONNECIONTS_ESTABLISHED.baseName());
+        addMetric(metrics, ContainerMetrics.JRT_TRANSPORT_CLIENT_TLS_CONNECTIONS_ESTABLISHED.baseName());
+        addMetric(metrics, ContainerMetrics.JRT_TRANSPORT_SERVER_UNENCRYPTED_CONNECTIONS_ESTABLISHED.baseName());
+        addMetric(metrics, ContainerMetrics. JRT_TRANSPORT_CLIENT_UNENCRYPTED_CONNECTIONS_ESTABLISHED. baseName());
 
         // C++ TLS metrics
-        addMetric(metrics, "vds.server.network.tls-handshakes-failed");
-        addMetric(metrics, "vds.server.network.peer-authorization-failures");
-        addMetric(metrics, "vds.server.network.client.tls-connections-established");
-        addMetric(metrics, "vds.server.network.server.tls-connections-established");
-        addMetric(metrics, "vds.server.network.client.insecure-connections-established");
-        addMetric(metrics, "vds.server.network.server.insecure-connections-established");
-        addMetric(metrics, "vds.server.network.tls-connections-broken");
-        addMetric(metrics, "vds.server.network.failed-tls-config-reloads");
+        addMetric(metrics, StorageMetrics.VDS_SERVER_NETWORK_TLS_HANDSHAKES_FAILED.count());
+        addMetric(metrics, StorageMetrics.VDS_SERVER_NETWORK_PEER_AUTHORIZATION_FAILURES.count());
+        addMetric(metrics, StorageMetrics.VDS_SERVER_NETWORK_CLIENT_TLS_CONNECTIONS_ESTABLISHED.count());
+        addMetric(metrics, StorageMetrics.VDS_SERVER_NETWORK_SERVER_TLS_CONNECTIONS_ESTABLISHED.count());
+        addMetric(metrics, StorageMetrics.VDS_SERVER_NETWORK_CLIENT_INSECURE_CONNECTIONS_ESTABLISHED.count());
+        addMetric(metrics, StorageMetrics.VDS_SERVER_NETWORK_SERVER_INSECURE_CONNECTIONS_ESTABLISHED.count());
+        addMetric(metrics, StorageMetrics.VDS_SERVER_NETWORK_TLS_CONNECTIONS_BROKEN.count());
+        addMetric(metrics, StorageMetrics.VDS_SERVER_NETWORK_FAILED_TLS_CONFIG_RELOADS.count());
```

Comment:
Is the change from base name to `.count` intended?

Reply:
Yes, intended. Should have been stated in the commit message, obviously. These have never been propagated to Yamas for this reason.

Reply:
👍